Documentation/qrexec-policy-agent.md

```diff
@@ -0,0 +1,34 @@
+# `qrexec-policy-agent`
+
+## Protocol
+
+The policy agent works as a socket-based service. It handles the following Qubes RPC calls:
+
+### `policy.Ask`
+
+Display a prompt about whether to allow an action.
+
+The request is a JSON dictionary with the following keys:
+- `source` - source domain (`"work"`)
+- `service_name` - name of service without argument (`"qubes.Filecopy"`)
```

Comment:
Okay, somehow I thought it's without argument because of [the discussion about notifications](https://github.com/QubesOS/qubes-issues/issues/3904#issuecomment-592516200). I will change that.

Reply:
Actually, that seems to be a regression in 4.1, introduced in 2018 with the new version of the tool. The old qrexec-policy [used the service_name argument directly](https://github.com/QubesOS/qubes-core-admin/blob/release4.0/qubespolicy/cli.py#L44), the new tool strips the argument during parsing.